tensorboard/uploader/proto/write_service.proto

```diff
@@ -119,6 +121,82 @@ message WriteScalarResponse {
   // This is empty on purpose.
 }
 
+// The blob writing procedure works as follows:
```

Comment:
Appreciate the super clear protocol docs. Thanks.

Reply:
Thanks!  Though I think I will need to update them in light of the comment threads. :)

Reply:
Updated substantially re BlobSequences and resumable uploads.